code/modules/asset_cache/assets/tgui.dm

```diff
@@ -1,3 +1,23 @@
+// If you use a file(...) object, instead of caching the asset it will be loaded from disk every time it's requested.
+// This is useful for development, but not recommended for production.
+// And if TGS is defined, we're being run in a production environment.
+
+#ifdef TGS
```

Comment:
seems a bit fair this will just be arbitrarily broken for anyone who isn't using TGS?

Reply:
I'd prefer it always be the case but AnturK is worried it'll break the dev server